test-framework/test-suites/integration/tests/dump/test_dump_network.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+import json
+
+@pytest.mark.usefixtures("revert_database")
```

Comment:
Don't need this decorator. Pytest is setup to always use the "revert_database" fixture.